README.md

```diff
@@ -1,85 +1,95 @@
 
-#### Librarified
-This library makes it super simple to do the Toopher two-step.  Check it out:
+### Step 1: Pair
+Before you can enhance your website's actions with Toopher, your customers will need to pair their mobile device's Toopher app with your website.  To do this, they generate a unique pairing phrase from within the app on their mobile device.  You will need to prompt them for a pairing phrase as part of the Toopher enrollment process.  Once you have a pairing phrase, just send it to the Toopher web service and we'll return a pairing ID that you can use whenever you want to authenticate an action for that user.
 
 ```python
 import toopher
 
-# Create an API object using your credentials
+# Create an API Object Using Your Credentials
 api = toopher.ToopherApi("<your consumer key>", "<your consumer secret>")
 
-# Step 1 - Pair with their phone's Toopher app
-pairing_status = api.pair("pairing phrase", "username@yourservice.com")
+# Step 1 - Pair with their mobile device's Toopher app
+pairing = api.pair("username@yourservice.com", "pairing phrase")
+```
+
+### Step 2: Authenticate
+You have complete control over what actions you want to authenticate using Toopher (for example: logging in, changing account information, making a purchase, etc.).  Just send us the user's pairing ID and a name for the terminal they're using and we'll make sure they actually want it to happen.
 
+```python
 # Step 2 - Authenticate a log in
-auth = api.authenticate(pairing_status.id, "my computer")
+authentication_request = api.authenticate(pairing.id, "terminal name")
 
 # Once they've responded you can then check the status
-auth_status = api.get_authentication_status(auth.id)
-if (auth_status.pending == False and auth_status.granted == True):
+authentication_request.refresh_from_server()
+if not authentication_request.pending and authentication_request.granted:
 	# Success!
 ```
 
-#### Handling Errors
-If any request runs into an error a `ToopherApiError` will be thrown with more details on what went wrong.
+## ToopherIframe Workflow
+### Step 1: Embed a request in an IFRAME
+1. Generate an authentication URL by providing a username.
+2. Display a webpage to your user that embeds this URL within an `<iframe>` element.
+
+```python
+import toopher
+
+# Create an API Object Using Your Credentials
+iframe_api = toopher.ToopherIframe("<your consumer key>", "<your consumer secret>")
+
 
-#### Zero-Storage usage option
-Requesters can choose to integrate the Toopher API in a way does not require storing any per-user data such as Pairing ID and Terminal ID - all of the storage
-is handled by the Toopher API Web Service, allowing your local database to remain unchanged.  If the Toopher API needs more data, it will `raise()` a specific
-error that allows your code to respond appropriately.
+auth_iframe_url = iframe_api.get_authentication_url(username);
+
+# Add an <iframe> element to your HTML:
+# <iframe id="toopher_iframe" src=auth_iframe_url />
```

Comment:
The plan was to have the [original HTML markup info](https://github.com/toopher/toopher-python/blob/develop/README-Iframe.md) moved to the dev site. Do you think it needs to be included here as well?